### ISSUE TITLE
Sonarwhal Website Lint Report for dwyl.com

### ISSUE THREAD

**Issue by nelsonic** [dwyl/dwyl-site#450]
https://sonarwhal.com is an _Open Source_ website scanning tool 
that checks for asite's accessibility, speed and common errors.
see: https://github.com/sonarwhal/sonarwhal

![image](https://user-images.githubusercontent.com/194400/38702094-9eda888e-3e97-11e8-843a-573ffa0b6e94.png)

The Scan for the dwyl.com website:
https://sonarwhal.com/scanner/17e38af9-0bb0-43f0-8d42-986c00151158
![image](https://user-images.githubusercontent.com/194400/38702106-a7f6ee26-3e97-11e8-865d-346479c90176.png)

Reveals quite a _few_ (86) errors ... 😞 
@iteles what are your thoughts on _addressing_ these?

**Comment by iteles** [comment]
That's interesting. We have used other tools in the past that have apparently been more 'forgiving':

See https://github.com/dwyl/dwyl-site/issues/14#issuecomment-334194425 and
http://wave.webaim.org/report#/dwyl.com
<img width="250" alt="screen shot 2018-04-12 at 22 27 47" src="https://user-images.githubusercontent.com/4185328/38705295-04795aea-3ea1-11e8-94c6-762c38c14ea6.png">

Also, not being able to have links inside `<li>`s? I've never even heard of that as a thing.

**Comment by nelsonic** [comment]
@iteles yeah, I was _not_ aware of the no `<a>` in `<li>` "rule" either ... 🙄 
I _like_ the fact that Sonarwhal can be run on Continuous Integration (Travis-CI) 🔴 / ✅ ?
such that any update to the site will _automatically_ be "_rejected_" if it fails to pass. ⛔️ 
Also, this goes _way_ beyond Accessibility ... 🔓 
_And_ it's Open Source so we can _inspect_ the code 🔍 
if/when we need to _understand_ the reason why something is failing ... 😉

**Comment by iteles** [comment]
Agreed! I had suggested we set up a pa11y pre-commit hook as a starter here: https://github.com/dwyl/dwyl-site/issues/409#issuecomment-334158068

**Comment by sohilpandya** [comment]
So I was lucky enough to see this one during the conference https://www.youtube.com/watch?v=O_OJfY4Jax4&feature=youtu.be

But we should share this video with everyone.

**Comment by iteles** [comment]
Oh I'm so glad there's a video of it @sohilpandya, thanks for sharing, the tweets from that session were FANTASTIC!

**Comment by nelsonic** [comment]
https://sonarwhal.com/
![image](https://user-images.githubusercontent.com/194400/195076175-3488e103-0821-446c-a7b9-30afb5744831.png)

Rebranded as https://webhint.io/
![image](https://user-images.githubusercontent.com/194400/195076312-10489c73-c559-4a58-9c54-8ecfe3b4fed2.png)

Still _super_ on-board with this. https://github.com/webhintio/hint
Definitely going to need _something_ like this for the **`new`** site: https://github.com/dwyl/dwyl-site/issues/493
